### PR TITLE
feat(runtime): configure pint runtime

### LINKED ISSUE #54: Configure Runtime with PINT pallets
## Issue summary

- Update the runtime configuration with all the pallets

## Other information and links

- Depends on #51

### PR DESCRIPTION
**Pull request by mattsse**
## Changes

- implements all PINT pallets for the runtime

## Tests

```
cargo test
```

## Issues

- Closes #54

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -426,6 +446,174 @@ impl pallet_local_treasury::Config for Runtime {
...
+     pub const MinCouncilVotes: usize = 4;
+}
+
+impl pallet_committee::Config for Runtime {
+    type ProposalSubmissionPeriod = ProposalSubmissionPeriod;
+    type VotingPeriod = VotingPeriod;
+    type MinCouncilVotes = MinCouncilVotes;
+    // Using root as the admin origin for now
+    type ProposalSubmissionOrigin = frame_system::EnsureRoot<AccountId>;
+    type ProposalExecutionOrigin = frame_system::EnsureRoot<AccountId>;
```
Comment (clearloop) [inline runtime/src/lib.rs:473]: We should open an issue for checking/organizing the config of `Origin` for pallets

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -426,6 +446,174 @@ impl pallet_local_treasury::Config for Runtime {
...
+}
+
+parameter_types! {
+    pub const FeedPalletId: PalletId = PalletId(*b"linkfeed");
+    pub const MinimumReserve: Balance = 100;
+    pub const StringLimit: u32 = 15;
+    pub const OracleLimit: u32 = 10;
+    pub const FeedLimit: u16 = 10;
+    pub const PruningWindow: u32 = 3;
+}
```
Comment (clearloop) [inline runtime/src/lib.rs:502]: Better to annotate docs for constants like these or linking to PINT docs?

**Review by clearloop — APPROVED** [review]
LGTM, for a reminder

* An issue for checking the origin config
* Docs for the constants in runtime

File: runtime/src/lib.rs (hunk 1 above)
Comment (mattsse) [reply runtime/src/lib.rs:473]: follow up on #64

**Pull request merged**